app/lib/activitypub/parser/media_attachment_parser.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class ActivityPub::Parser::MediaAttachmentParser
+  include JsonLdHelper
+
+  def initialize(json)
+    @json = json
+  end
+
+  # @param [MediaAttachment] other
+  def ==(other)
+    remote_url == other.remote_url &&
+      thumbnail_remote_url == other.thumbnail_remote_url &&
+      description == other.description
+  end
```

Comment:
I'm not super comfortable with overriding `==`, especially since in practice you end up using it for two different object types. I'd prefer if it were a custom method instead.

Also, it doesn't cover `file_content_type`, `focus` nor `blurhash`, which is probably fine in the context where you use it, but even more of a reason for it not to be `==`.